tools/gitHooks/prettier.js

```diff
@@ -1,60 +0,0 @@
-/**
- * Copyright 2017 Google Inc.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *   http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-const { resolve } = require('path');
-const { spawn } = require('child-process-promise');
-const fs = require('mz/fs');
-const glob = require('glob');
-const simpleGit = require('simple-git/promise');
-const ora = require('ora');
```

Comment:
We won't need it necessarily no. It can always be run manually by just calling `./node_modules/.bin/prettier` and I'd imagine we do so fairly rarely.

Reply:
Agreed.

Reply:
Yeah...  I more meant "Do we still need a script that runs prettier on the whole repo?" since the commit hooks aren't going to catch everything (rebases, amend commits, post-prettier-upgrade reformats, etc.)?

Maybe it's simple enough to just run it from the command-line manually though. I'm not sure what all this script was doing...

Reply:
This script just runs prettier on the whole SDK and gives you progress updates. So we could probably just make an NPM script in the repo root to take care of the global style hammer.